docs/zkapps/o1js/on-chain-values.mdx

```diff
@@ -64,13 +64,6 @@ class VotingApp extends SmartContract {
 
 A more refined example could store the current start date in an on-chain state variable, which can then be reset by some process that is also encoded by the zkApp.
 
-In addition to using a predefined range, you can also construct a range that depends on another variable, such as the current time. For example, a DEX with an order book could support orders that expire after an hour. In this case, the range would be `[now, now + 1h]`:
```

Comment:
this example was never needed to convey that "you can also construct a range that depends on another variable". The sentence right above it, about storing the start date on the zkApp, is already an excellent example use case for constructing a range from a variable